test/e2e/apimachinery/aggregator.go

```diff
@@ -33,7 +33,9 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/apiserver/pkg/authentication/serviceaccount"
+	utilversion "k8s.io/kubernetes/pkg/util/version"
+	//"k8s.io/apiserver/pkg/authentication/serviceaccount"
```

Comment:
Removed all "dead" code. It will be coming back once I get a chance to fix the TODO.